english/sci-fi-dungeon/1/the-end.md

```diff
@@ -0,0 +1,7 @@
+**You fought with pride.**
+
+You are dead ☠️.
+
+**THE END 🎬**
+
+[Restart the game](../begin-journey.md)
```

Comment:
1. A dungeon has a lot of ending floor, we could not create a step that its name is `the end`, due to depending on the story we could have different `the end`.
2. If doesn't have another room reference for this file, so please delete this file.

Reply:
yes sure. no problem. 😄

Reply:
hi ming, can you do a little favor on me by adding hacktoberfest-accepted tag on this PR, please as i have to complete 4 PRs this month.
Tysm.